### PR TITLE
1.18.x/interact me

### PR DESCRIPTION
**Pull request by mrsterner**
cherrypicked all the 1.20 stuff down here, and since vs2 1.18 have support tor patch f it also builds :woman_dancing:

### CODE REVIEW

**Comment by mrsterner** [comment]
Triooode :angry: 
@StewStrong

**Comment by ewoudje** [comment]
Closes https://github.com/ValkyrienSkies/Create-Interactive-Issues/issues/74

**Comment by mrsterner** [comment]
this hot and heavy pr snoring right now

File: build.gradle [hunk 1]
```diff
@@ -43,6 +43,10 @@ subprojects {
             name = "ParchmentMC"
             url = "https://maven.parchmentmc.org"
         }
+        maven { url = "https://mvn.devos.one/snapshots/" }
+        maven { url = "https://mvn.devos.one/releases/" }
+        maven { url = "https://raw.githubusercontent.com/Fuzss/modresources/main/maven/" }
+        maven { url 'https://www.jitpack.io' }
```
Comment (StewStrong) [inline build.gradle]: jitpack bad

**Review by StewStrong — APPROVED** [review]

**Pull request merged**